### PR TITLE
[JAVA] Fixed tests, add more methods

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

**Comment by likholat** [comment]
@dkurt

File: modules/java_api/tests/TensorTests.java [hunk 1]
```diff
@@ -5,11 +5,22 @@
...
     public void testGetTensorFromFloat() {
-        int[] dimsArr = {1, 1, 2, 2};
-        float[] data = {0.0f, 1.1f, 2.2f, 3.3f};
+        int size = 1;
+        for (int val : dimsArr) size *= val;
+
+        float[] data = new float[size];
+
+        Random rand = new Random();
+        for (int i = 0; i < size; i++) data[i] = rand.nextFloat();
```
Comment (dkurt) [inline modules/java_api/tests/TensorTests.java:21]: It's better to keep using fixed data, not random.

File: modules/java_api/tests/PrePostProcessorTests.java [hunk 2]
```diff
@@ -17,15 +16,12 @@ public class PrePostProcessorTests extends OVTest {
...
         core = new Core();
         net = core.read_model(modelXml);
-        float[] data = {
-            0.0f, 1.1f, 2.2f, 3.3f, 4.4f, 5.5f, 6.6f, 7.7f, 8.8f, 9.9f, 1.1f, 2.2f, 3.3f
-        };
+        float[] data = {0.0f, 1.1f, 2.2f, 3.3f, 4.4f, 5.5f, 6.6f, 7.7f, 8.8f, 9.9f, 1.1f, 2.2f};
         input = new Tensor(dimsArr, data);
     }
 
-    @Ignore
```
Comment (dkurt) [inline modules/java_api/tests/PrePostProcessorTests.java:26]: Do you know if this test became stable? Of not, let's enable ot by a separate PR

Comment (likholat) [reply modules/java_api/tests/PrePostProcessorTests.java:26]: Yes, I think the problem was about `TensorFloat` method

**Comment by likholat** [comment]
@allnes

File: modules/java_api/tests/TensorTests.java (hunk 1 above)
Comment (dkurt) [reply modules/java_api/tests/TensorTests.java]: Randomizer take more resources. Also, data volume os small and we can keep it explicitly initialized.

**Review by dkurt — APPROVED** [review]
LGTM

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**